requirements.txt

```diff
@@ -0,0 +1,10 @@
+pip>=9
+setuptools>=26
+wheel>=0.29
+pytest
+pandas
+numpy
+dash
+dash-core-components 
+dash-html-components
+plotly
```

Comment:
Better version these requirements (at least a lower bound) ;)

Reply:
I added the versions I usually use as a lower bound